stuff/cron/aggregate_feedback.py

```diff
@@ -35,15 +35,25 @@
                                     LEFT JOIN `User_Rank` as ur
                                     ON ur.`user_id` = qn.`user_id`
                                     WHERE `nomination` = 'suggestion'
-                                      AND qn.`qualitynomination_id` > %s;"""
+                                      AND qn.`qualitynomination_id` > %s
+                                      AND qn.`qualitynomination_id` IN (
+                                          SELECT MAX(qualitynomination_id)
+                                          FROM `QualityNominations`
+                                          GROUP BY user_id, problem_id
```

Comment:
qué tal

```suggestion
                                      AND qn.`qualitynomination_id` = (
                                          SELECT MAX(qualitynomination_id)
                                          FROM `QualityNominations`
                                          WHERE
                                              `user_id` = qn.`user_id` AND
                                              `problem_id` = qn.`problem_id`
```

? así es más claro que buscas la última nominación para _ese_ usuario/problema.